### DISCUSSION TITLE
Trace http and grpc body

### DISCUSSION THREAD

**Discussion by Volodymyr-Maiboroda-Imprivata** [open-telemetry/opentelemetry-java-instrumentation#2336] [Q&A]
Hi guys, I'm trying to use opentelemetry javaagent in our application. We need to trace HTTP and grpc bodies. 
Is there a possible way to configure it?

**Comment by trask** [comment]
hi @Volodymyr-Maiboroda-Imprivata! check out #1317, https://github.com/open-telemetry/opentelemetry-specification/issues/1062 and https://github.com/open-telemetry/opentelemetry-specification/issues/1284 (cc: @pavolloffay)
